src/algebra/group_power/linear_ordered_field.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2015 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import algebra.group_power.lemmas
+import data.nat.cast
+/-!
+# Lemmas about power operations on fields
```

Comment:
Can this go in `algebra.order.field_pow` as well?

Reply:
Yes!